.travis.yml

```diff
@@ -1,19 +1,18 @@
 language: node_js
-branches:
-  only:
-    - master
-    - /^greenkeeper/.*$/
 cache:
-  yarn: true
   directories:
-    - node_modules
+    - ~/.npm
```

Comment:
See https://stackoverflow.com/questions/42521884/should-i-have-travis-cache-node-modules-or-home-npm